Dockerfile

```diff
@@ -1,93 +1,72 @@
-# Dockerfile to build an image able to compile flashlight for a variety of
-# platforms.
-#
-# > make docker
+# This docker machine is able to compile and sign Lantern for Linux and
+# Windows.
 
 FROM fedora:21
-MAINTAINER "José Carlos Nieto" <xiam@getlantern.org>
+MAINTAINER "The Lantern Team" <team@getlantern.org>
 
-ENV GO_VERSION go1.4.3
-ENV GOROOT_BOOTSTRAP /go1.4
-ENV GOROOT /go
-ENV GOPATH /
-
-ENV PATH $PATH:$GOROOT/bin
 ENV WORKDIR /lantern
+ENV SECRETS /secrets
 
-# Go binary for bootstrapping.
-ENV GO_PACKAGE_URL https://storage.googleapis.com/golang/go1.4.3.linux-amd64.tar.gz
+RUN mkdir -p $WORKDIR
+RUN mkdir -p $SECRETS
 
 # Updating system.
-RUN yum -y update && yum clean all
+RUN yum update -y && yum makecache && yum clean packages
+RUN yum install -y deltarpm && yum clean packages
```

Comment:
I found that `deltarpm` only works for `yum update`, so we need to change these two lines as below to have it take effect.

```
RUN yum install -y deltarpm && yum update -y && yum clean packages
```

It reduced 50 M of updates to 16 M (67% saved), not a significant save considering lots of `yum install` hereafter, but we can also just keep it.

I had no knowledge of `makecache` but seems we benefit litle from it as we didn't supply `-C` option to `yum install` (ref [Red Hat doc](https://access.redhat.com/documentation/en-US/Red_Hat_Enterprise_Linux/6/html/Deployment_Guide/sec-Working_with_Yum_Cache.html)).